src/main/java/net/neoforged/accesstransformer/AccessTransformerEngineImpl.java

```diff
@@ -11,13 +10,10 @@
 import org.objectweb.asm.tree.MethodNode;
 
 import java.io.IOException;
+import java.io.Reader;
 import java.net.URISyntaxException;
 import java.nio.file.Path;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.Spliterator;
-import java.util.Spliterators;
+import java.util.*;
```

Comment:
No wildcard imports.

Reply:
AT already has a good number. Can we set up spotless in another PR maybe?

Reply:
Fixed them here instead